docs/system_overview/configuration.md

```diff
@@ -25,6 +25,9 @@ This file defines the behavior of the server Agent that answers individual reque
         // Changing this value requires a server restart in order to take effect.
         "agent_factory_watcher_sleep_time_in_seconds": 5,
 
+        // The read buffer size in bytes used for checksum calculations of a data object at rest.
```

Comment:
Consider rewording this. Here's one suggestion:
```
// The buffer size used for reading a replica during checksum calculations. 
```

If this property is optional, put `// (Optional)` on its own line like so:
```
// (Optional)
// The buffer size used for reading a replica during checksum calculations. 
```